app/models/post.rb

```diff
@@ -4,6 +4,7 @@ class Post < ActiveRecord::Base
   has_many :comments, dependent: :destroy
   has_many :labelings, as: :labelable
   has_many :labels, through: :labelings
+  has_many :ratings, as: :rateable
```

Comment:
Look back at the [assignment](https://www.bloc.io/users/brenda-schmidt/checkpoints/1648?roadmap_section_id=120). A Rating has many Posts, not the other way around. Sorry!